frontend/__tests__/components/pod.spec.tsx

```diff
@@ -48,3 +48,59 @@ describe(PodsDetailsPage.displayName, () => {
     expect(wrapper.find(DetailsPage).exists()).toBe(true);
   });
 });
+
+describe('Renders ContainerRow', () => {
+  let pod;
+  let container;
+  let wrapper: ShallowWrapper;
+  beforeEach(() => {
+    pod = {
+      status: {
+        conditions: [],
+        containerStatuses: [],
+      },
+    };
+    container = {
+      name: 'hello-openshift',
+      image: 'aosqe/hello-openshift',
+    };
+  });
+
+  it('renders container Name in row', () => {
+    wrapper = shallow(<ContainerRow pod={pod} container={container} />);
+    expect(wrapper.find('ContainerLink').find({name: 'hello-openshift'}).exists()).toBe(true);
+  });
+
+  it('renders container Image in row', () => {
+    wrapper = shallow(<ContainerRow pod={pod} container={container} />);
+    expect(wrapper.find('div').find({className: 'col-lg-2 col-md-3 col-sm-5 col-xs-7 co-truncate co-nowrap co-select-to-copy'}).text()).toContain('aosqe/hello-openshift');
```

Comment:
I don't think we should select on these classes. It would be better use to column index (here and below).